src/Rules/PhpDoc/InvalidThrowsPhpDocValueRule.php

```diff
@@ -48,6 +49,9 @@ public function processNode(Node $node, Scope $scope): array
 		}
 
 		$phpDocThrowsType = $resolvedPhpDoc->getThrowsTag()->getType();
+		if ((new VoidType())->accepts($phpDocThrowsType, $scope->isDeclareStrictTypes())->yes()) {
```

Comment:
It's better to use `isSuperTypeOf`, `accepts` is only for a few special places like checking argument types.

Reply:
Ok, changed